queries/http/highlights.scm

```diff
@@ -1,19 +1,39 @@
-; Schemes
-(scheme) @type
+; Keywords
+(scheme) @module
 
 ; Methods
 (method) @function.method
 
 ; Constants
 (const_spec) @constant
 
+; Headers
+(header
+  name: (name) @constant)
+
 ; Variables
-(identifier) @variable
+(variable_declaration
+  name: (identifier) @variable)
+
+(variable_declaration
+  value: (number))
+
+(variable_declaration
+  value: (boolean))
+
+(variable_declaration
+  value: (string))
```

Comment:
You don't need to write this anymore if there isn't any captures with it

Reply:
Done, thanks 😅